packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -21,8 +21,25 @@ export const outgoingStateToString = {
   [OutgoingMessageState.EXECUTED]: 'Executed'
 }
 
-export const getDepositStatus = (tx: Transaction) => {
-  if (tx.type !== 'deposit' && tx.type !== 'deposit-l1') return undefined
+export const getDepositStatus = (tx: Transaction | MergedTransaction) => {
+  const transaction = tx as Transaction
```

Comment:
We need to get rid of these type casts. We can have utils that check if something is `Transaction` and `MergedTransaction`

```
  if (isMergedTransaction(tx)) {
  } else {
  }
```

Reply:
updated